examples/using-MDX/gatsby-ssr.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Implement Gatsby's SSR (Server Side Rendering) APIs in this file.
+ *
+ * See: https://www.gatsbyjs.org/docs/ssr-apis/
+ */
```

Comment:
Same with the other 2 API files: this one can go away if unused to keep things simple.